buildSrc/src/main/kotlin/Testing.kt

```diff
@@ -81,12 +81,9 @@ class NessieTestingPlugin : Plugin<Project> {
       }
       if (plugins.hasPlugin("io.quarkus")) {
         // This directory somehow disappears... Maybe some weird Quarkus code.
-        tasks.named("quarkusGenerateCodeTests") {
```

Comment:
Such changes elimniate the use of script references, which do not work with Gradle's configuration cache (not serializable)